cms/djangoapps/contentstore/courseware_index.py

```diff
@@ -163,7 +177,9 @@ def index_item(item, skip_index=False):
                 item_index['id'] = item_id
                 if item.start:
                     item_index['start_date'] = item.start
-
+                item_index['content_groups'] = item_content_groups
+                if item_index['content_groups'] == []:
```

Comment:
Nit: This can now go in one line
`item_index['content_groups'] = item_content_groups if item_content_groups else None`